rpm_spec/gpg-split-dom0.spec.in

```diff
@@ -19,25 +19,11 @@
 # You should have received a copy of the GNU General Public License
 # along with this program; if not, write to the Free Software
 # Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
-#
-#
-
-# enable python3 by default
-%bcond_without python3
-
-# python2 is not available on RHEL > 7 and not needed on Fedora > 28
-%if 0%{?rhel} > 7 || 0%{?fedora} > 28
-# disable python2 by default
-%bcond_with python2
-%else
-%bcond_without python2
-%endif
-
```

Comment:
If you remove this bcond, please cleanup its usage in the spec too. The way you did it currently made no python files installed at all (neither 2, nor 3).

Reply:
…oops